app/controllers/hippo/diffs_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Hippo::DiffsController < Comfy::Admin::Cms::BaseController
+  before_action :check_admin
+
+  def show
+    @hippo_diff = Cms::HippoDiff.new(data: data)
+  end
+
+  private
+
+  def data
+    File.read(Rails.root.join('tmp', filename))
```

Comment:
Again, I will write in case you didn't read the comment on the PR. The perfect solution will be scraping the content xml file on the Hippo CMS System, instead expecting on the tmp dir. Will do this in a separate PR.